app/templates/gentelella/admin/event/wizard/basic-details.html

```diff
@@ -204,7 +204,7 @@
                        value="External Event URL" readonly style="background-color: white;">
             </div>
             <div class="col-sm-5 input-group" v-bind:class="{ 'has-error': !isValidLinkEntry(event.event_url) }">
-                <input type="url" class="form-control" v-model="event.event_url" placeholder="Link">
+                <input type="url" class="form-control" v-model="event.event_url" attribute name="event_url" placeholder="Link">
```

Comment:
@niranjan94 I added the name attribute since I found that all input files had name to it. But it was my mistake to write attribute aswell. Should I remove it?